gis/mod_tile/Portfile

```diff
@@ -120,9 +115,9 @@ if {![variant_isset postgresql15] && ![variant_isset postgresql16]} {
     default_variants-append +postgresql16
 }
 
-post-extract {
-    file rename ${worksrcpath}/renderd.conf ${worksrcpath}/renderd.conf.orig
-}
+# post-extract {
+#     file copy ${worksrcpath}/etc/renderd/renderd.conf ${worksrcpath}/etc/renderd/renderd.conf.orig
+# }
```

Comment:
don't comment lines out; if it's not needed anymore just remove

Reply:
Removed